BizHawk.Client.Common/movie/tasproj/IStateManager.cs

```diff
@@ -7,16 +7,29 @@ namespace BizHawk.Client.Common
 	public interface IStateManager
 	{
 		// byte[] this[int frame] { get; } // TODO: I had it refactored to this back in the day
+
+		/// <summary>
+		/// Retrieves the savestate for the given frame,
+		/// If this frame does not have a state currently, will return an empty array
+		/// </summary>
+		/// <returns>A savestate for the given frame or an empty array if there isn't one</returns>
```

Comment:
`returns` just repeats `summary`, looks unnecessary.

Reply:
"correct" documentation is supposed to be like that.
The returns shows in a different situation than the summary sometimes and stuff